lib/stackify-api-ruby.rb

```diff
@@ -10,7 +10,7 @@ module Stackify
   INTERNAL_LOG_PREFIX = '[Stackify]'.freeze
   STATUSES = { working: 'working', terminating: 'terminating', terminated: 'terminated'}
   MODES = { logging: :logging, metrics: :metrics, both: :both }
-  TRANSPORT = [DEFAULT = 'default', UNIX_SOCKET = 'agent_socket', AGENT_HTTP = 'agent_http']
```

Comment:
why the rename from *default* to *log* ?

Reply:
Just changed back from log to default options.